das/stats.go

```diff
@@ -5,10 +5,12 @@ package das
 // over current network headers, and the `catchUp` routine which performs sampling
 // over past headers from the last sampled checkpoint.
 type SamplingStats struct {
-	// all headers before SampledBefore were successfully sampled
-	SampledBefore uint64 `json:"sampled_before_height"`
-	// MaxKnown is the height of the newest known header
-	MaxKnown uint64 `json:"max_known_height"`
+	// all headers before HeadOfSampledChain were successfully sampled
+	HeadOfSampledChain uint64 `json:"head_of_sampled_chain"`
+	// all headers before HeadOfCatchup were submitted to sampling workers
+	HeadOfCatchup uint64 `json:"head_of_catchup"`
+	// NetworkHead is the height of the most recent header in the network
+	NetworkHead uint64 `json:"network_head_height"`
```

Comment:
CatchupHead and SampledChainHead and more consistent with NetworkHead imo

Reply:
renamed